internal/cli/apps.go

```diff
@@ -12,6 +12,9 @@ import (
 	"gopkg.in/auth0.v5/management"
 )
 
+// ErrNoApps signifies no applications exist in a tenant
+var ErrNoApps = errors.New("there are currently no applications")
```

Comment:
Only thing I'd say is private by default if we don't think other packages will use this error.

Reply:
Good call, spaced that everything is in cli package